components/mpas-ocean/src/shared/mpas_ocn_tendency.F

```diff
@@ -1411,7 +1411,7 @@ subroutine ocn_tend_freq_filtered_thickness(tendPool, statePool, &
             iEdge = edgesOnCell(i, iCell)
 
             do k = 1, maxLevelEdgeTop(iEdge)
-               flux = layerThickEdge(k,iEdge)*normalVelocity(k,iEdge)* &
+               flux = layerThickEdgeFlux(k,iEdge)*normalVelocity(k,iEdge)* &
```

Comment:
@dengwirda Pointing you to the use of `layerThickEdgeFlux` in `ocn_tend_tracer` for your review.

Reply:
I don't think this one is in `ocn_tend_tracer`, it's in `ocn_tend_freq_filtered_thickness`.  But it still deserves a careful look.

Reply:
Good catch. @dengwirda Feel free to ignore :)